inductiva/tasks/task.py

```diff
@@ -278,6 +278,46 @@ class Task:
         models.TaskStatusCode.PENDINGINPUT, models.TaskStatusCode.STARTED
     }
 
+    STATUS_MESSAGES = {
+        models.TaskStatusCode.PENDINGINPUT:
+            "Task %s was created and is waiting for the required input files "
+            "to be uploaded.",
+        models.TaskStatusCode.EXECUTERTERMINATED:
+            "The machine running your task was terminated by the cloud provider"
+            " and your task stopped running.\nUse the `resubmit_on_preemption`"
+            " parameter in the Python client to configure if the task should be"
```

Comment:
where in the python client?
Shouldn't it be more informative and say that it is when submitting the task?

Reply:
```
            "The machine running your task was terminated by the cloud provider"
            " and it stopped.\nUse the `resubmit_on_preemption` parameter when "
            "submitting the task to configure it to automatically resubmitted "
            "to the queue.",
```

Not sure if i should use "submitting the task to configure it to automatically resubmitted " or "running the task to configure it to automatically resubmitted "